controllers/hyperconverged/hyperconverged_controller_test.go

```diff
@@ -714,7 +714,7 @@ var _ = Describe("HyperconvergedController", func() {
 				).To(BeNil())
 
 				Expect(foundResource.Status.RelatedObjects).ToNot(BeNil())
-				Expect(len(foundResource.Status.RelatedObjects)).Should(Equal(18))
+				Expect(len(foundResource.Status.RelatedObjects)).Should(Equal(17))
```

Comment:
is this due to the prometheus Rule referencing the deployment?
maybe we should still list it under relatedObjects

Reply:
Fixed. I added the PrometheusRule to the relatedObjects list, but the implementation is in the `alerts` package, so it won't affect this test.